### PR TITLE
[MOV] accounting: move customer addresses from sales to accounting

### PR DESCRIPTION
**Pull request by dade-odoo**
task-3304975

### CODE REVIEW

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Configuration
+=============
+
+To specify a sales order's invoice and delivery addresses, first go to :menuselection:`Accounting
+--> Configuration --> Settings`. In the :guilabel:`Customer Invoices` setion, enable
+:guilabel:`Customer Addresses`. On quotations and sales orders, there are now fields for
+:guilabel:`Invoice Address` and :guilabel:`Delivery Address`. If the customer has an invoice or
+delivery address listed on their contact record, the corresponding field will use that address by
+default, but any contact's address can be used instead.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:20]:
```suggestion
To specify a sales order's invoice and delivery addresses, first go to :menuselection:`Accounting
app --> Configuration --> Settings`. In the :guilabel:`Customer Invoices` setion, enable
:guilabel:`Customer Addresses`, and click :guilabel:`Save` to save all changes.

Now, on quotations and sales orders, there are now fields for :guilabel:`Invoice Address` and
:guilabel:`Delivery Address`. If the customer has an invoice or delivery address listed on their
contact record, the corresponding field uses that address by default, but any contact's address
can be used instead.
```

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Invoice and deliver to different addresses
+==========================================
+
+Delivery orders and their delivery slip reports use the address set as the :guilabel:`Delivery
+Address` on the sales order. Invoice reports by default show both shipping address and invoice
+address to assure the customer that the delivery will go to the correct address.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:27]:
```suggestion
Address` on the sales order. By default, invoice reports show both, the shipping address and the
invoice address, to assure the customer that the delivery will go to the correct location.
```

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Emails also go different addresses. The quotation and sales order are sent to the main contact's
+email as usual, but the invoice is instead sent to the email of the address set as the
+:guilabel:`Invoice Address` on the sales order.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:31]:
```suggestion
Emails also go to different addresses. The quotation and sales order are sent to the main contact's
email, as usual, but the invoice is sent to the email of the address set as the :guilabel:`Invoice
Address` on the sales order.
```

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+.. note::
+   - Reports such as the delivery slip and invoice report can be :doc:`customized using Studio
+     </applications/productivity/studio/pdf_reports>`.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:35]:
```suggestion
   - Reports, such as the delivery slip and invoice report, can be :doc:`customized using Studio
     </applications/productivity/studio/pdf_reports>`.
```

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+   - If :doc:`Send by Post <snailmail>` is checked when you :guilabel:`Send & Print`, the invoice
+     will be mailed to the invoice address.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:37]:
```suggestion
   - If :doc:`Send by Post <snailmail>` is checked when :guilabel:`Send & Print` is selected, the
   invoice is mailed to the invoice address.
```

**Review by ksc-odoo — COMMENTED** [review]
Hey @dade-odoo - great work on this doc. Only a few minor modifications needed, but once those are implemented, I think it's ready to move on to the next stage 👍

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst (hunk 5 above)
Comment (ksc-odoo) [reply content/applications/finance/accounting/customer_invoices/customer_addresses.rst:37]: Make sure to leave one blank line at the end of all docs

Comment (dade-odoo) [reply content/applications/finance/accounting/customer_invoices/customer_addresses.rst:37]: Ah right - thanks!

**Review by jcs-odoo — APPROVED** [review]
Okay from a technical pov 👍 

@robodoo r+

**Pull request closed without merging**